sdks/csharp/.github/workflows/publish-nuget.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish NuGet Package on Release
```

Comment:
Ok this is quite nice now that when we send out a SpacetimeDB release it will also release the nuget package for us automatically. This is a huge improvement.

Reply:
Yeah I guess it's not true in the general case that we *always* want to send out C# SDK updates for every SpacetimeDB release (especially for minor version bumps)